splinepy/helpme/extract.py

```diff
@@ -681,12 +692,13 @@ def spline(self, splitting_plane=None, interval=None):
 
         Parameters
         ----------
-        splitting_plane : int / dictionary (int : (float))
-          if integer : parametric dimension to be extracted
-          if dictionary : list of splitting planes and ranges to be passed
-        interval : float / tuple<float,float>
+        splitting_plane : int or dictionary {int : (float, float))
```

Comment:
Was it your intend to change the first brace? If it was you should match also the last brace.
```suggestion
        splitting_plane : int or dictionary {int : (float, float)}
```

Reply:
nice catch